packages/webview_flutter/webview_flutter_platform_interface/lib/v4/src/platform_navigation_delegate.dart

```diff
@@ -9,6 +9,19 @@ import 'package:plugin_platform_interface/plugin_platform_interface.dart';
 
 import 'webview_platform.dart';
 
+/// Signature for callbacks that report a pending navigation request.
+typedef NavigationRequestCallback = FutureOr<bool> Function(
```

Comment:
Instead of returning `bool`, could this return [NavigationDecision](https://pub.dev/documentation/webview_flutter/latest/webview_flutter/NavigationDecision.html)?

Reply:
Definitely, that is indeed more explicit. Done.